archive/g/go/string-reverse.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import "fmt"
+
+func reverse_string(str string) string {
+	chars := []rune(str)
+	for i, j := 0, len(chars)-1; i < j; i, j = i+1, j-1 {
+		chars[i], chars[j] = chars[j], chars[i]
+	}
+	return string(chars)
+}
+
+func main() {
+	fmt.Printf("%v\n", reverse_string("hacktoberfest"))
```

Comment:
Can you pull the string from the command line?

Reply:
Also, can you verify that there's a string on the command line to grab. For example:

```
if args.length > 0:
    // print reversed string
else:
    // print error/usage instructions
```

Reply:
I will update it.

Reply:
Hey @riddhik84! Did you ever get around to this?

Reply:
Working on it. I will update the code today!